ote/src/clj/ote/tasks/gtfs.clj

```diff
@@ -66,9 +66,9 @@
 ;; To speed up detection, call this with vector of service-ids:
 ;; e.g.: (detect-new-changes-task db (time/now) true [1289])
```

Comment:
Could you update this example how this can be called from repl? Something like (def config (:config (ote.main/config))

Reply:
Thanks, done